src/Data/Managers/GetInstanceWithRelations/QueryManager.ts

```diff
@@ -0,0 +1,140 @@
+import { useContext, useEffect } from "react";
+import {
+  ApiHelper,
+  Query,
+  Either,
+  RemoteData,
+  ServiceInstanceModel,
+  StateHelperWithEnv,
+  OneTimeQueryManager,
+} from "@/Core";
+import { Type } from "@/Core/Language/Either";
+import { DependencyContext } from "@/UI";
+
+type Data = RemoteData.Type<
+  Query.Error<"GetInstanceWithRelations">,
+  Query.UsedData<"GetInstanceWithRelations">
+>;
+
+export function GetInstanceWithRelationsQueryManager(
+  apiHelper: ApiHelper,
+  stateHelper: StateHelperWithEnv<"GetInstanceWithRelations">
+): OneTimeQueryManager<"GetInstanceWithRelations"> {
+  function getInstanceUrl({
+    id,
+  }: Query.SubQuery<"GetInstanceWithRelations">): string {
+    return `/lsm/v1/service_inventory?service_id=${id}&include_deployment_progress=false&exclude_read_only_attributes=false`;
+  }
+
+  function getInstanceWithRelationsUrl({
+    service_entity,
+    id,
+  }: Query.SubQuery<"GetServiceInstance">): string {
+    return `/lsm/v1/service_inventory/${service_entity}/${id}?include_deployment_progress=false&exclude_read_only_attributes=false&include_referenced_by=true`;
+  }
+
+  function initialize(
+    query: Query.SubQuery<"GetInstanceWithRelations">,
+    environment
+  ): void {
+    const value = stateHelper.getOnce(query, environment);
+    if (RemoteData.isNotAsked(value)) {
+      stateHelper.set(RemoteData.loading(), query, environment);
+    }
+  }
+
+  async function update(
+    query: Query.SubQuery<"GetInstanceWithRelations">,
+    environment: string
+  ): Promise<void> {
+    const instance = await getInstanceWithRelations(query, environment);
+    //stateHelper implementation interfaces want to get direct api response which is transformed,
+    //decided to set is a any to avoid having separate implementation just for this call
+    stateHelper.set(
+      RemoteData.fromEither(
+        instance as unknown as Type<string, { data: ServiceInstanceModel }>
```

Comment:
If it's temporary, can you make a ticket and link the ticket in the comment ? :D

Reply:
I'll add that in the tests PR, and that ticket will be connected to the queryManager V2,